[bitnami/harbor] failed to install harbor, harbor-core can't access  Redis

### Name and Version

Bitnami/harbor

### What steps will reproduce the bug?

1,vcd+tkgm
kubernetes:1.20.14
harbor chart: harbor-16.0.0
app version: 2.6.1

### Are you using any custom parameters or values?

```yaml
core:
  ## Bitnami Harbor Core image
  ## ref: https://hub.docker.com/r/bitnami/harbor-core/tags/
  ## @param core.image.registry Harbor Core image registry
  ## @param core.image.repository Harbor Core image repository
  ## @param core.image.tag Harbor Core image tag (immutable tags are recommended)
  ## @param core.image.digest Harbor Core image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
  ## @param core.image.pullPolicy Harbor Core image pull policy
  ## @param core.image.pullSecrets Harbor Core image pull secrets
  ## @param core.image.debug Enable Harbor Core image debug mode
  ##
  image:
    registry: docker.io
    repository: bitnami/harbor-core
    tag: 2.6.0-debian-11-r20
    digest: ""
    ## Specify a imagePullPolicy
    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
    ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-images
    ##
    pullPolicy: IfNotPresent
    ## Optionally specify an array of imagePullSecrets.
    ## Secrets must be manually created in the namespace.
    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
    ## e.g:
    ## pullSecrets:
    ##   - myRegistryKeySecretName
    ##
    pullSecrets: []
    ## Enable debug mode
    ##
    debug: false

  ## @param core.sessionLifetime Explicitly set a session timeout (in seconds) overriding the backend default.
  ##
  sessionLifetime: ""
  ## @param core.uaaSecret If using external UAA auth which has a self signed cert, you can provide a pre-created secret containing it under the key `ca.crt`.
  ##
  uaaSecret: ""
  ## @param core.secretKey The key used for encryption. Must be a string of 16 chars
  ## e.g:
  ## secretKey: "not-a-secure-string"
  ##
  secretKey: ""
  ## @param core.secret Secret used when the core server communicates with other components. If a secret key is not specified, Helm will generate one. Must be a string of 16 chars.
  ##
  secret: ""
  ## @param core.secretName Fill the name of a kubernetes secret if you want to use your own TLS certificate and private key for token encryption/decryption. The secret must contain two keys named: `tls.crt` - the certificate and `tls.key` - the private key. The default key pair will be used if it isn't set
  ##
  secretName: ""
  ## @param core.csrfKey The CSRF key. Will be generated automatically if it isn't specified
  ##
  csrfKey: ""
  ## Use TLS in the container
  ##
  ##
  tls:
    ## @param core.tls.existingSecret Name of an existing secret with the certificates for internal TLS access
    ## Requires `internalTLS.enabled` to be set to `true``
    ## Self-signed TLS certificates will be used otherwise
    ##
    existingSecret: ""
  ## @param core.command Override default container command (useful when using custom images)
  ##
  command: []
  ## @param core.args Override default container args (useful when using custom images)
  ##
  args: []
  ## @param core.extraEnvVars Array with extra environment variables to add Harbor Core pods
  ##
  extraEnvVars: []
  ## @param core.extraEnvVarsCM ConfigMap containing extra environment variables for Harbor Core pods
  ##
  extraEnvVarsCM: ""
  ## @param core.extraEnvVarsSecret Secret containing extra environment variables (in case of sensitive data) for Harbor Core pods
  ##
  extraEnvVarsSecret: ""
  ## @param core.configOverwriteJson String containing a JSON with configuration overrides
  ## Source: https://goharbor.io/docs/latest/install-config/configure-user-settings-cli/#harbor-user-settings
  ##
  configOverwriteJson: ""
  ## @param core.configOverwriteJsonSecret Secret containing the JSON configuration overrides
  ## Source: https://goharbor.io/docs/latest/install-config/configure-user-settings-cli/#harbor-user-settings
  ##

redis:
  enabled: true
  volumePermissions:
    enabled: true
    image:
      registry: docker.io
      repository: bitnami/bitnami-shell
      tag: 11-debian-11-r48
    containerSecurityContext:
      runAsUser: 0
  auth:
    enabled: true
    ## Redis&reg; password (both master and slave). Defaults to a random 10-character alphanumeric string if not set and auth.enabled is true.
    ## It should always be set using the password value or in the existingSecret to avoid issues
    ## with Harbor.
    ## The password value is ignored if existingSecret is set
    ##
    password: "********"
    existingSecret: ""
  architecture: standalone
```

### What is the expected behavior?

<img width="676" alt="image" src="https://user-images.githubusercontent.com/3224084/201258233-7ae28cc6-38d2-4f9d-b9c7-475228cd6afb.png">


<img width="1440" alt="image" src="https://user-images.githubusercontent.com/3224084/201258777-af6a2e01-48b4-4ab1-922b-dae4795f0bb8.png">



### What do you see instead?

Redid password had been set, but harbor-core access redid using redis:xxxxx. How to change the password in harbor-core

### Additional information

_No response_

When I set auth to true, and leave password blank, there is an error in harbor-core:
<img width="726" alt="image" src="https://user-images.githubusercontent.com/3224084/202595837-1d280622-5128-4c70-bf63-8907beeda924.png">


```
2022-11-18T01:23:30Z [INFO] [/core/main.go:136]: initializing cache ...
2022-11-18T01:23:30Z [ERROR] [/lib/cache/cache.go:114]: failed to ping redis://bitnami-harbor-redis-master:6379/0, retry after 500ms : NOAUTH Authentication required.
2022-11-18T01:23:30Z [ERROR] [/lib/cache/cache.go:114]: failed to ping redis://bitnami-harbor-redis-master:6379/0, retry after 616.711996ms : NOAUTH Authentication required.
2022-11-18T01:23:31Z [ERROR] [/lib/cache/cache.go:114]: failed to ping redis://bitnami-harbor-redis-master:6379/0, retry after 575.41182ms : NOAUTH Authentication required.
2022-11-18T01:23:32Z [ERROR] [/lib/cache/cache.go:114]: failed to ping redis://bitnami-harbor-redis-master:6379/0, retry after 3.959319865s : NOAUTH Authentication required.
2022-11-18T01:23:36Z [ERROR] [/lib/cache/cache.go:114]: failed to ping redis://bitnami-harbor-redis-master:6379/0, retry after 6.686287704s : NOAUTH Authentication required.
2022-11-18T01:23:42Z [ERROR] [/lib/cache/cache.go:114]: failed to ping redis://bitnami-harbor-redis-master:6379/0, retry after 10s : NOAUTH Authentication required.
2022-11-18T01:23:52Z [ERROR] [/lib/cache/cache.go:114]: failed to ping redis://bitnami-harbor-redis-master:6379/0, retry after 10s : NOAUTH Authentication required.
2022-11-18T01:24:02Z [ERROR] [/lib/cache/cache.go:114]: failed to ping redis://bitnami-harbor-redis-master:6379/0, retry after 10s : NOAUTH Authentication required.
```

--- a/bitnami/harbor/Chart.yaml
+++ b/bitnami/harbor/Chart.yaml
@@ -36,4 +36,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/harbor-registry
   - https://github.com/bitnami/containers/tree/main/bitnami/harbor-registryctl
   - https://goharbor.io/
-version: 16.4.8
+version: 16.5.0

--- a/bitnami/harbor/Chart.yaml
+++ b/bitnami/harbor/Chart.yaml
@@ -36,4 +36,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/harbor-registry
   - https://github.com/bitnami/containers/tree/main/bitnami/harbor-registryctl
   - https://goharbor.io/
-version: 16.5.0
+version: 16.4.9

--- a/bitnami/harbor/templates/_helpers.tpl
+++ b/bitnami/harbor/templates/_helpers.tpl
@@ -263,13 +263,13 @@ Return whether Redis&reg; uses password authentication or not
 {{- define "harbor.redisForJobservice" -}}
   {{- if and (eq .Values.externalRedis.sentinel.enabled false) (eq .Values.redis.sentinel.enabled false) -}}
     {{- if (include "harbor.redis.escapedRawPassword" . ) -}}
-      {{- printf "redis://redis:%s@%s:%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.jobserviceDatabaseIndex" . ) -}}
+      {{- printf "redis://default:%s@%s:%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.jobserviceDatabaseIndex" . ) -}}
     {{- else -}}
       {{- printf "redis://%s:%s/%s" (include "harbor.redis.host" .) (include "harbor.redis.port" .) (include "harbor.redis.jobserviceDatabaseIndex" .) -}}
     {{- end -}}
   {{- else -}}
     {{- if (include "harbor.redis.escapedRawPassword" . ) -}}
-      {{- printf "redis+sentinel://redis:%s@%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.jobserviceDatabaseIndex" . ) -}}
+      {{- printf "redis+sentinel://default:%s@%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.jobserviceDatabaseIndex" . ) -}}
     {{- else -}}
       {{- printf "redis+sentinel://%s/%s" (include "harbor.redis.host" . ) (include "harbor.redis.jobserviceDatabaseIndex" . ) -}}
     {{- end -}}
@@ -280,13 +280,13 @@ Return whether Redis&reg; uses password authentication or not
 {{- define "harbor.redisForGC" -}}
   {{- if and (eq .Values.externalRedis.sentinel.enabled false) (eq .Values.redis.sentinel.enabled false) -}}
     {{- if (include "harbor.redis.escapedRawPassword" . ) -}}
-      {{- printf "redis://redis:%s@%s:%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.registryDatabaseIndex" . ) -}}
+      {{- printf "redis://default:%s@%s:%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.registryDatabaseIndex" . ) -}}
     {{- else -}}
       {{- printf "redis://%s:%s/%s" (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.registryDatabaseIndex" . ) -}}
     {{- end -}}
   {{- else -}}
     {{- if (include "harbor.redis.escapedRawPassword" . ) -}}
-      {{- printf "redis+sentinel://redis:%s@%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.registryDatabaseIndex" . ) -}}
+      {{- printf "redis+sentinel://default:%s@%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.registryDatabaseIndex" . ) -}}
     {{- else -}}
       {{- printf "redis+sentinel://%s/%s" (include "harbor.redis.host" . ) (include "harbor.redis.registryDatabaseIndex" . ) -}}
     {{- end -}}
@@ -296,13 +296,13 @@ Return whether Redis&reg; uses password authentication or not
 {{- define "harbor.redisForTrivyAdapter" -}}
   {{- if and (eq .Values.externalRedis.sentinel.enabled false) (eq .Values.redis.sentinel.enabled false) -}}
     {{- if (include "harbor.redis.escapedRawPassword" . ) -}}
-      {{- printf "redis://redis:%s@%s:%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.trivyAdapterDatabaseIndex" . ) -}}
+      {{- printf "redis://default:%s@%s:%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.trivyAdapterDatabaseIndex" . ) -}}
     {{- else -}}
       {{- printf "redis://%s:%s/%s" (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.trivyAdapterDatabaseIndex" . ) -}}
     {{- end -}}
   {{- else -}}
     {{- if (include "harbor.redis.escapedRawPassword" . ) -}}
-      {{- printf "redis+sentinel://redis:%s@%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.trivyAdapterDatabaseIndex" . ) -}}
+      {{- printf "redis+sentinel://default:%s@%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.trivyAdapterDatabaseIndex" . ) -}}
     {{- else -}}
       {{- printf "redis+sentinel://%s/%s" (include "harbor.redis.host" . ) (include "harbor.redis.trivyAdapterDatabaseIndex" . ) -}}
     {{- end -}}
@@ -312,13 +312,13 @@ Return whether Redis&reg; uses password authentication or not
 {{- define "harbor.redisForCore" -}}
   {{- if and (eq .Values.externalRedis.sentinel.enabled false) (eq .Values.redis.sentinel.enabled false) -}}
     {{- if (include "harbor.redis.escapedRawPassword" . ) -}}
-      {{- printf "redis://redis:%s@%s:%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.coreDatabaseIndex" . ) -}}
+      {{- printf "redis://default:%s@%s:%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.coreDatabaseIndex" . ) -}}
     {{- else -}}
       {{- printf "redis://%s:%s/%s" (include "harbor.redis.host" . ) (include "harbor.redis.port" . ) (include "harbor.redis.coreDatabaseIndex" . ) -}}
     {{- end -}}
   {{- else -}}
     {{- if (include "harbor.redis.escapedRawPassword" . ) -}}
-      {{- printf "redis+sentinel://redis:%s@%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.coreDatabaseIndex" . ) -}}
+      {{- printf "redis+sentinel://default:%s@%s/%s" (include "harbor.redis.escapedRawPassword" . ) (include "harbor.redis.host" . ) (include "harbor.redis.coreDatabaseIndex" . ) -}}
     {{- else -}}
       {{- printf "redis+sentinel://%s/%s" (include "harbor.redis.host" . ) (include "harbor.redis.coreDatabaseIndex" . ) -}}
     {{- end -}}

